audit/var-154.md

```diff
@@ -0,0 +1,35 @@
+# Chain Variable Transaction 154
+
+## Changes
+
+This transaction sets `security_reward_bugfix` to `true` and
```

Comment:
does this want an explanation of any kind or is the var commented in the var header file?

Reply:
updated in 2e23b21